wagtail/admin/templates/wagtailadmin/shared/field_as_li.html

```diff
@@ -0,0 +1,2 @@
+{% comment %}RemovedInWagtail60 - this template is no longer used in Wagtail but kept around a little longer for compatibility with third-party packages.{% endcomment %}
```

Comment:
The corresponding Python class is `RemovedInWagtail60Warning`, let's use the same name to be consistent and to make it easier when we look for it later.

```suggestion
{% comment %}
RemovedInWagtail60Warning - this template is no longer used in Wagtail but kept around until the next major release to avoid a breaking change.
{% endcomment %}
```

Could you add the upgrade considerations as well? Thanks!

Reply:
I already added upgrade considerations when I merged in the original PR. But double check the wording.

Reply:
Note. We use this string, without the word warning in lots of other places like all JS future deprecations.